crates/runtime/src/container.rs

```diff
@@ -48,7 +48,7 @@ pub async fn start(
     network: &str,
     task_name: &str,
     task_type: ops::TaskType,
-    allow_local: bool,
+    publish_ports: bool,
```

Comment:
`allow_local` was a confusing name in this context, so I updated it to reflect what it actually does.